src/views/home.hbs

```diff
@@ -1,8 +1 @@
-
-<div id="search">
-  <form class="search__form">
-     <input class="search__input" type="text" placeholder="Search by
-        keyword">
-     <input class="search__button" type="submit">Search</input>
-  </form>
-</div>
+{{>searchbar}}
```

Comment:
what im thinking is that this is view that has 1 partial now, so i hope that theres content gonna be added later on

Reply:
@Karyum yup there will be more partials added later on!